package-lock.json

```diff
@@ -4908,7 +5044,7 @@
         },
         "load-json-file": {
             "version": "1.1.0",
-            "resolved": "https://registry.npmjs.org/load-json-file/-/load-json-file-1.1.0.tgz",
+            "resolved": "http://registry.npmjs.org/load-json-file/-/load-json-file-1.1.0.tgz",
```

Comment:
You're having that https to http problem again. Make sure to follow the steps https://github.com/npm/npm/issues/20106#issuecomment-383770315